Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -227,11 +227,11 @@ public bool UseLinkedAltScaleOverride
         }
 
         [SerializeField]
-        [Tooltip("Raycast direction. Can cast from head in facing direction, or cast from head to object position")]
-        private RaycastDirectionMode currentRaycastDirectionMode = RaycastDirectionMode.ToLinkedPosition;
+        [Tooltip("Raycast direction type. Default is forward direction of Tracked Target transform")]
+        private RaycastDirectionMode currentRaycastDirectionMode = RaycastDirectionMode.TrackedTargetForward;
```

Comment:
>TrackedTargetForward [](start = 88, length = 20)

What's the reason for changing the default mode of this solver?

This has a risk of breaking someone in a really hard to debug way... though I guess technically if they've already serialized these scenes it should have saved the old default value.

Reply:
@wiwei I actually want to consider looking at removing linkedposition lol. It is the default and it doesn't work. It generates a raystep of 0,0,0 to 0,0,0. 

This shouldn't break customers though because the linkedposition enum is still there if they selected it. Only new components being added will be impacted which is a smaller set of customers...who again I hope were switching it anyways to get it to work. 

I actually know of one customer who may have been dynamically doing it. I'll have to check